main/nodejs/APKBUILD

```diff
@@ -83,11 +82,12 @@ build() {
 		--shared-zlib \
 		--shared-libuv \
 		--shared-openssl \
-		--shared-http-parser \
+		--experimental-http-parser \
```

Comment:
experimental doesn't sound nice, we can't use an un-experimental version? :/

Reply:
Good catch, the new ("experimental") parser lhttp should be the default with Node.js v12: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#semver-major-commits - I'll update the PR

Reply:
Done